src/software/ai/hl/stp/tactic/tactic_interface.h

```diff
@@ -0,0 +1,66 @@
+#pragma once
+
+#include "software/ai/hl/stp/tactic/tactic_visitor.h"
+/*
+ * Interface for Tactics, so that we can make a std::vector<TacticInterface>
+ * Otherwise, we would have to use a template which doesn't really work.
+ */
+
+class TacticInterface
```

Comment:
I'd suggest rewriting this comment since it's not really necessary to explain why we need an interface separate from the template base class (it's just good OOP design) and the use case for this interface is much more general than just `std::vector`. In fact, I would rename `TacticInterface` to `Tactic` and make this header comment about what Tactics are in our codebase, similar to the [description here](https://github.com/UBC-Thunderbots/Software/blob/master/docs/software-architecture-and-design.md#tactics)